local_cache/local_cache.go

```diff
@@ -506,7 +506,7 @@ func (sc *LocalCache) runMux() error {
 
 			sourceURL := *sc.directorURL
 			sourceURL.Path = path.Join(sourceURL.Path, path.Clean(req.request.path))
-			tj, err := sc.tc.NewTransferJob(&sourceURL, localPath, false, false, client.WithToken(req.request.token))
+			tj, err := sc.tc.NewTransferJob(&sourceURL, localPath, false, false, "", client.WithToken(req.request.token))
```

Comment:
It would be a nice idea here to say the "project" is the localcache

Reply:
Can do!